src/main/kotlin/net/irisshaders/lilybot/api/pluralkit/PluralKit.kt

```diff
@@ -96,6 +136,7 @@ data class PluralKitMessage(
 	val timestamp: Instant,
 	val id: Snowflake,
 	val original: Snowflake,
+	val sender: Snowflake,
```

Comment:
Would it not make more sense for this to be author?

Reply:
I'm following the API documentation, it would, but I'd rather not differ from the spec